### ISSUE TITLE
Bug: TypeError: Cannot read property '0' of undefined

### ISSUE THREAD

**Issue by deman4ik** [fullstack-build/tslog#7]
**Describe the bug**
2020-06-04 15:13:57.858  INFO   [test.ts:5]     TypeError: Cannot read property '0' of undefined
    at stylizeWithColor (internal/util/inspect.js:340:27)
    at formatPrimitive (internal/util/inspect.js:1120:12)
    at formatValue (internal/util/inspect.js:533:12)
    at formatProperty (internal/util/inspect.js:1406:11)
    at formatRaw (internal/util/inspect.js:785:9)
    at formatValue (internal/util/inspect.js:591:10)
    at Object.inspect (internal/util/inspect.js:221:10)
    at ../node_modules/tslog/src/index.ts:435:26
    at Array.forEach (<anonymous>)
    at Logger._printPrettyLog (../node_modules/tslog/src/index.ts:430:30)

**To Reproduce**
```ts
import { Logger } from "tslog";

const logger = new Logger();

logger.info({ foo: "bar" });
```

**Expected behavior**
Log object

**Context**
Node v12.14.0
ts-node v8.10.2
tslog v2.1.0

**Comment by terehov** [comment]
Hey mate, 

thanks for pointing it out and providing such a detailed bug report. 
It seems to be an issue inside of that particular Node version (12.14 & 12.5)`at stylizeWithColor (internal/util/inspect.js:340:27)` and has been fixed with 12.16 already.

I am still investigating, so maybe I'll be able to come up with a workaround. However, you may want to upgrade to 12.16 in the meantime :-)

**Comment by terehov** [comment]
@deman4ik So it turned out to be a problem with some `bright` colors like `blueBright`, `redBright`. I checked the versions in question (12.14) and it works now, however not as colorful as on all the other versions < 12 and >= 12.16.

Thanks again for pointing it out.
